hardhat.config.js

```diff
@@ -40,7 +40,7 @@ const argv = require('yargs/yargs')()
     compiler: {
       alias: 'compileVersion',
       type: 'string',
-      default: '0.8.13',
+      default: '0.8.18',
```

Comment:
We may want to return this back to `0.8.13` and discuss in another PR, I would personally prefer to check it more carefully.